src/instance.rs

```diff
@@ -131,38 +146,154 @@ impl<T> Py<T> {
+    //         PyObject::from_owned_ptr_or_err(py, ffi::PyObject_GetAttr(self.as_ptr(), attr_name))
+    //     })
+    // }
+
+    // /// Calls the object.
+    // ///
+    // /// This is equivalent to the Python expression `self(*args, **kwargs)`.
+    // pub fn call(
+    //     &self,
+    //     py: Python,
+    //     args: impl IntoPy<Py<PyTuple>>,
+    //     kwargs: Option<&PyDict>,
+    // ) -> PyResult<PyObject> {
+    //     let args = args.into_py(py).into_ptr();
+    //     let kwargs = kwargs.into_ptr();
+    //     let result = unsafe {
+    //         PyObject::from_owned_ptr_or_err(py, ffi::PyObject_Call(self.as_ptr(), args, kwargs))
+    //     };
+    //     unsafe {
+    //         ffi::Py_XDECREF(args);
+    //         ffi::Py_XDECREF(kwargs);
+    //     }
+    //     result
+    // }
+
+    // /// Calls the object with only positional arguments.
+    // ///
+    // /// This is equivalent to the Python expression `self(*args)`.
+    // pub fn call1(&self, py: Python, args: impl IntoPy<Py<PyTuple>>) -> PyResult<PyObject> {
+    //     self.call(py, args, None)
+    // }
+
+    // /// Calls the object without arguments.
+    // ///
+    // /// This is equivalent to the Python expression `self()`.
+    // pub fn call0(&self, py: Python) -> PyResult<PyObject> {
+    //     self.call(py, (), None)
+    // }
+
+    // /// Calls a method on the object.
+    // ///
+    // /// This is equivalent to the Python expression `self.name(*args, **kwargs)`.
+    // pub fn call_method(
+    //     &self,
+    //     py: Python,
+    //     name: &str,
+    //     args: impl IntoPy<Py<PyTuple>>,
+    //     kwargs: Option<&PyDict>,
+    // ) -> PyResult<PyObject> {
+    //     name.with_borrowed_ptr(py, |name| unsafe {
+    //         let args = args.into_py(py).into_ptr();
+    //         let kwargs = kwargs.into_ptr();
+    //         let ptr = ffi::PyObject_GetAttr(self.as_ptr(), name);
+    //         if ptr.is_null() {
+    //             return Err(PyErr::fetch(py));
+    //         }
+    //         let result = PyObject::from_owned_ptr_or_err(py, ffi::PyObject_Call(ptr, args, kwargs));
+    //         ffi::Py_DECREF(ptr);
+    //         ffi::Py_XDECREF(args);
+    //         ffi::Py_XDECREF(kwargs);
+    //         result
+    //     })
+    // }
+
+    // /// Calls a method on the object with only positional arguments.
+    // ///
+    // /// This is equivalent to the Python expression `self.name(*args)`.
+    // pub fn call_method1(
+    //     &self,
+    //     py: Python,
+    //     name: &str,
+    //     args: impl IntoPy<Py<PyTuple>>,
+    // ) -> PyResult<PyObject> {
+    //     self.call_method(py, name, args, None)
+    // }
+
+    // /// Calls a method on the object with no arguments.
+    // ///
+    // /// This is equivalent to the Python expression `self.name()`.
+    // pub fn call_method0(&self, py: Python, name: &str) -> PyResult<PyObject> {
+    //     self.call_method(py, name, (), None)
+    // }
```

Comment:
These methods used to exist on `PyObject`. I'm trying to decide if we should move them to `Py<T>`, or delete them.